BUILD.gn

```diff
@@ -388,11 +393,21 @@ static_library("electron_lib") {
   if (is_win) {
     sources += filenames_gypi.lib_sources_win
   }
+  if (is_mac && is_mas_build) {
```

Comment:
It should be enough to check `is_mas_build` here.
`is_mas_build == true` doesn't make any sense on non-Mac.